src/plugins/intel_cpu/src/compiled_model.h

```diff
@@ -13,12 +14,13 @@
 #include "openvino/runtime/iinfer_request.hpp"
 #include "openvino/runtime/iplugin.hpp"
 #include "openvino/runtime/isync_infer_request.hpp"
-#include "openvino/runtime/threading/thread_local.hpp"
 #include "sub_memory_manager.hpp"
 
 namespace ov {
 namespace intel_cpu {
 
+class NetworkMemoryControl;
+
```

Comment:
Looks like this forward declaration is also not needed anymore.

Reply:
Removed